requirements.txt

```diff
@@ -1,2 +1,3 @@
 esphome>=2023.12.0
 pre-commit>=3.4.0
+tomli>=2.0.1
```

Comment:
what's this for?

Reply:
Some doc indicated you needed tomli for ruff to parse your config files but now I can't find that anywhere; removing it.